interfaces/IAnalytics.h

```diff
@@ -0,0 +1,39 @@
+#pragma once
+
+#include "Module.h"
+
+// @stubgen:include <com/IIteratorType.h>
+
+namespace WPEFramework {
+namespace Exchange {
+    // @json 1.0.0
+    struct EXTERNAL IAnalytics : virtual public Core::IUnknown {
+        enum { ID = ID_ANALYTICS };
+
+        virtual ~IAnalytics() override = default;
```

Comment:
No need for virtual and override are mutual exclusive. Use one or the other. So just remove virtual here.

Reply:
Fixed